trigger.sh

```diff
@@ -146,14 +179,23 @@ fi
 
 result=`echo $?`
```

Comment:
The placement of this 'result' is peculiar to me.  Since there are so many 'if' blocks, the last command could have been line 177, 170, or line 157.  What is the intent of this check (and subsequent validation on lines 180-186)?

If you are checking the pip install result, it needs to be moved prior to line 173.

Reply:
Here I am trying to ensure that "**python3 prerequisite_changes.py ${test_service} ${target}**" runs without any error. So I will move this to line 176.